docs/gradle-plugin/migrations/migration-to-0.8.0.md

```diff
@@ -0,0 +1,144 @@
+For JVM projects, nothing changes compared to the `0.7.x` version, however, for Android projects, 
+running one of these tasks will result running tests  for all build variants present in the project.
+
+### Custom reports variants
+In the `0.7.x` version, it was allowed to merge a report for several Android build variants only into default reports,
+it was not possible to create several combinations with the arbitrary inclusion of different build variants.
+
+Since `0.8.0` in order to merge several Android build variants, need to create custom reports variant.
+
+The newly created variant is initially empty, in order for classes to appear in the report and tests to be executed 
+when generating it, need to add existing variants provided from Kotlin targets to it: it can be a `jvm` variant, or any of an Android build variant.
+```kotlin
+kover {
+    variants {
+        create("custom") {
+            add("jvm")
+            add("release")
+        }
+    }
+}
+```
+
+Creating a variant with a name `custom` will add a task named `koverHtmlReportCustom`, `koverXmlReportCustom`, `koverVerifyCustom`, `koverLogCustom`, `koverBinaryReportCustom`.
+
+### More about the reports variant
+A reports variant is a set of information used to generate a report, namely: 
+project classes, a list of Gradle test tasks, classes that need to be excluded from instrumentation.
+
+There are several types of report variants: 
+- total variant (have no special name)
+- variant for classes in JVM target (named `jvm`)
+- variants for the Android build variant (the name matches the name of the build variant)
+- custom variants (the name is specified when creating)
+
+For each variant, a set of tasks is created in the project to generate reports on the information contained in it.
+
+The names are generated according to the following rule:
+`koverHtmlReport<VariantName>`, `koverXmlReport<VariantName>`, `koverVerify<VariantName>`, `koverLog<VariantName>`, `koverBinaryReport<VariantName>`
+
+
+The reports variant can be used in another project to create a merged reports.
+To do this, in another project, which we will call the merging project, we will specify a dependency on the project from which we want to import classes and Gradle test tasks.
+```kotlin
+dependencies {
+    kover(project(":lib"))
+}
+```
+As a result, if you run the `:koverHtmlReport` task, it will run all the tests from merging project and `:lib` project 
+and generate a report for all classes from merging project and `:lib` project.
+
+However, if you call a task for a named variant, for example `:koverHtmlReportRelease`, then it will run tests for `release` variant from merging project and `:lib` project
+and generate a report for classes of `release` variant from merging project and `:lib` project.
+
+At the same time, it is recommended that variant `release` is also present in the `:lib` project, however, 
+for technical reasons, such a check may not be implemented in `0.8.0` and subsequent versions.
```

Comment:
Error that the Kover artifact with attribute `kover-variant=release`cannot be resolved in the `:lib` project

Reply:
So there is an error? Then what 'such a check may not be implemented' refers to? It means that we cannot provide a nice error message instead of Gradle generic one with 'attribute cannot be resolved'?

Reply:
Not in all cases there will be an error. In some cases, there will be an explicit error (as I remember in Android without different flavors), and in other cases it will simply be ignored